helix-stdx/src/env.rs

```diff
@@ -110,15 +110,15 @@ fn expand_impl(src: &OsStr, mut resolve: impl FnMut(&OsStr) -> Option<OsString>)
             continue;
         }
         let var = &bytes[captures.get_group(1).unwrap().range()];
-        let default = if pattern_id != 5 {
+        let default = if pattern_id == 5 {
+            &[]
```

Comment:
I disagree with most of these. The main code block should be larger code block, since that's the main case, and the else case is kind of like a fallback. !is_empty() is just used as an equivalent of.is_some() in these conditionals.